_installation/single-multiple.md

```diff
@@ -26,16 +33,59 @@ CREATE USER babelfish_user WITH CREATEDB
 	CREATEROLE PASSWORD '<PUT_SECRET_PASS_HERE>' INHERIT;
 DROP DATABASE IF EXISTS demo;
 
+/* the Babelfish database */
 CREATE DATABASE demo OWNER babelfish_user;
+ALTER SYSTEM SET babelfishpg_tsql.database_name = 'demo';
+SELECT pg_reload_conf();
+
+/* determine the migration mode */
+ALTER DATABASE demo SET babelfishpg_tsql.migration_mode = 'single-db'|'multi-db';
 
 \c demo
 
 CREATE EXTENSION IF NOT EXISTS "babelfishpg_tds" CASCADE;
-ALTER SYSTEM SET babelfishpg_tsql.database_name = 'demo';
-ALTER DATABASE demo SET babelfishpg_tsql.migration_mode = 'single-db'|'multi-db';
 CALL SYS.INITIALIZE_BABELFISH('babelfish_user');
 ```
 
-The `ALTER DATABASE` statement assigns the correct parameter to the database.
-The init function will then perform all the steps needed to perform the setup.
+The `ALTER DATABASE` statement assigns the correct migration mode to the
+database.  The `SYS.INITIALIZE_BABELFISH` function will then perform all
+the steps needed to set up Babelfish.
+
+Once you are connected to the database via TDS, you can utilize the `USE`
+command to select the current database as you would do it in Microsoft
+SQL Server (with `single-db`, you can have only a single database).
```

Comment:
You cna have only a single DB at a time: when you drop it you can create another one

Reply:
Well, that's understood. I don't think that picking nits like that will make the documentation more accessible.